news/2025/2025-04-15-the-followpoint-azer-the-tournament-titan.md

```diff
@@ -0,0 +1,305 @@
+
+[OWC 2018 vs Australia: First pick strategy](https://youtu.be/xGXobUU6wdY?si=yQjhdUfAI4AxqPUi&t=592) (This video has corrupt audio near the end of the first map, sorry!)
+
+This one requires some context, but I felt like I had to include this moment as a captain and not as a player. We had a really tough matchup against team Australia, who this year (and ever since) was arguably the strongest speed/DT roster of the entire tournament. We knew for SURE that they would win the 2 remaining picks of that type we could not ban out, so instead of letting that happen, I came up with the idea of grabbing the first pick and preying on tournament nerves being at their highest on match start. This was 100% lost in normal conditions and everyone in my team knew this. We essentially stole one of their best picks and forced them to play it at the worst possible conditions, while the rest of our team just had fun **assuming that the pick would be lost**. As it turns out, when you play with no nerves with the expectation of losing, you end up playing pretty well, so we somehow won that pick due to this strategy. It was extremely crucial in the match and took us to tiebreaker, which we also ended up winning. This is definitely still the best tournament strategy I came up with to this day!
+
+<a class="avatar news-chat-quote__avatar" href="https://osu.ppy.sh/users/18152711" style="background-image: url('/wiki/shared/avatars/MegaMix.jpg')"></a>
+
+<p class="news-chat-quote__username"><a class="news-chat-quote__colour-lvd" href="https://osu.ppy.sh/users/18152711">MegaMix</a></p>
+
+[Did we kick a kid?](https://osu.ppy.sh/beatmaps/artists/200) :tf:
+
+<a class="avatar news-chat-quote__avatar" href="https://osu.ppy.sh/users/2155578" style="background-image: url('/wiki/shared/avatars/Azer.jpg')"></a>
+
+<p class="news-chat-quote__username"><a class="news-chat-quote__colour-gmt" href="https://osu.ppy.sh/users/2155578">Azer</a></p>
+
+You know, I don't think this part of the meme comes from me or my community, I could be wrong but I really don't remember any presence associated with it. I think it came later from someone else getting a play on it or the other mapset, maybe. I don't really mind it though, since people mostly link it to the song. Funny story, I made friends with a new player somewhat recently and they were curious about the history of the game and various figures within it, so eventually they asked who I was, and the answer they got from their friend was simply that "I kick kids". No other context was ever given. This is my legacy, I guess..?
```

Comment:
```suggestion
You know, I don't think this part of the meme comes from me or my community, I could be wrong but I really don't remember any presence associated with it. I think it came later from someone else getting a play on it or the other mapset, maybe. I don't really mind it, though, since people mostly link it to the song. Funny story, I made friends with a new player somewhat recently and they were curious about the history of the game and various figures within it, so eventually they asked who I was, and the answer they got from their friend was simply that "I kick kids". No other context was ever given. This is my legacy, I guess..?
```